src/table.rs

```diff
@@ -75,7 +75,7 @@ pub const KEY_LEN: usize = 32;
 pub const SIZE_TIERS: usize = 1usize << SIZE_TIERS_BITS;
 pub const SIZE_TIERS_BITS: u8 = 8;
 pub const COMPRESSED_MASK: u16 = 0x80_00;
-pub const MAX_ENTRY_SIZE: usize = 0x7ff8;
+pub const MAX_ENTRY_SIZE: usize = 0x7ff8; // Actual max size in V4 was 0x7dfe
```

Comment:
Also found 0x7dfe for v4, but for new bigger entry size I am not really sure.
Should it be 7ffc: '7f' to account for compression byte and 'fc' as first value bellow any of tombstone, multipart or multihead.